expertise/service/expertise.py

```diff
@@ -43,9 +43,9 @@ def __init__(self, client, config, logger):
         self.mfr_checkpoint_dir = config['MFR_CHECKPOINT_DIR']
 
         # Define expected/required API fields
-        self.req_fields = ['name', 'match_group', 'paper_invitation', 'user_id', 'job_id']
+        self.req_fields = ['name', 'match_group', 'user_id', 'job_id']
         self.optional_model_params = ['use_title', 'use_abstract', 'average_score', 'max_score', 'skip_specter']
-        self.optional_fields = ['model', 'model_params', 'exclusion_inv', 'token', 'baseurl']
+        self.optional_fields = ['model', 'model_params', 'exclusion_inv', 'token', 'baseurl', 'paper_invitation', 'paper_id']
```

Comment:
I'm assuming one of either `paper_invitation` or `paper_id` is required, right @haroldrubio? If yes, add a check for that in `validate_config`.